areas/acr/src/AzureMcp.Acr/AcrSetup.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using AzureMcp.Acr.Commands.Registry;
+using AzureMcp.Acr.Services;
+using AzureMcp.Core.Areas;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace AzureMcp.Acr;
+
+public class AcrSetup : IAreaSetup
+{
+    public void ConfigureServices(IServiceCollection services)
+    {
+        services.AddSingleton<IAcrService, AcrService>();
+    }
+
+    public void RegisterCommands(CommandGroup rootGroup, ILoggerFactory loggerFactory)
+    {
+        var acr = new CommandGroup("acr", "Azure Container Registry operations - Commands for managing Azure Container Registry resources. Includes operations for listing container registries and managing registry configurations.");
```

Comment:
This is not a question on this PR, but a general question.

When initially adding `CommandGroup`s is the expectation for the description to be everything that will be added into the `CommandGroup` or should it be kept to what is actually supported in that command group and updated each time new tools are added?

Reply:
I'd prefer to keep it generic and not list all features.